src/protocols/secure_channel/StatusReport.cpp

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
nit: may want to add chip license blurb and remove this @file comment (it is obvious that `StatusReport.cpp` `implements methods for the StatusReport class`)

Reply:
done